mirrord/layer/src/macros.rs

```diff
@@ -91,17 +91,18 @@ macro_rules! hook_symbol {
 
 #[macro_export]
 macro_rules! graceful_exit {
-    ($($arg:tt)+) => {
+    ($($arg:tt)+) => {{
```

Comment:
Missing something, why the double parentheses?

Reply:
So that a call to that macro can be used as its own arm on `match`. Without it we would have to wrap calls to it in curly brackets, because without them we would be putting multiple statements directly in a `match` arm which is not allowed.